### DISCUSSION TITLE
Powered USB OTG Adapter

### DISCUSSION THREAD

**Discussion by wbvczar** [feelfreelinux/octo4a#47] [Q&A] [answered]
I just loaded OctoPrint for Android up on an old Droid Turbo 2 running Android 7.0.  I'm so excited about this and I did get it working on the phone.

It was recommended in the article to get a Powered USB OTG Adapter (link to Amazon).  I purchased the adapter but I found out it does not charge the phone.  I looked closer at the item description and it says "PLEASE NOTE: It can NOT charge any Phone or Tablet at the same time, so it will ONLY support OTG if you buy it for Phone or Tablet".

I looked at other adapters like this and that seems to be the norm.  Has anyone found something that will charge the phone (micro USB) and do OTG to the 3D printer?

This is an incredible option from the developer, so I hope there is another option.

**Comment by keymaster777** [comment, marked as answer]
AFAIK the only cable splitter that can charge with OTG is USB-C (so your phone needs a usbc port, converters won't make it work). There are tutorials out there showing how to rig your phone to charge with otg if the battery is removable by adding a "charger" in the battery access.

**Reply by Meguinness** [reply]
This one will charge the phone that as a micro usb, 
Switch the button to Charge...
Connect power to the micro usb female using your phone charger..
Connect  your phone using the micro usb  ...
Switch the button to OTG..
Connect printer HDMI cable to any of the 3 HDMI sockets.

This works 👍

Micro Usb HUB Adaptor with Power, TUSITA 3-Port Charging OTG Host Cable Cord Adapter for TV Stick, Raspberry Pi 2 3 Pi Zero Android Smart Phone Tablet https://www.amazon.co.uk/dp/B078M3Z84Z/ref=cm_sw_r_apan_glt_fabc_MDGGM9090MP52D5TEGG5?_encoding=UTF8&psc=1

**Comment by sbaeder** [comment]
look harder on Amazon...I found this [one](https://smile.amazon.com/s?k=Powered-Adapter-Charging-Streaming-Android%2F)... (search for "Powered-Adapter-Charging-Streaming-Android"

**Reply by Blaze350454** [reply]
do you mean the little 90s. they say will not charge phone. do you have it and it works?

**Reply by Meguinness** [reply]
https://www.amazon.co.uk/dp/B078M3Z84Z/ref=cm_sw_r_apan_glt_fabc_MDGGM9090MP52D5TEGG5?_encoding=UTF8&psc=1

**Comment by RobyTill** [comment]
I found this page where there is a schematic for making a powered OTG cable.
[https://blog.workshoptwelve.com/2019/01/04/otg-usb-charge-hub-wtf/](url)

A ready-made product is also indicated but I would like to try to build it.
Has anyone tried it?
Thanks
